testfiles/ASHRAE901_HotelLarge_STD2019_Denver.idf

```diff
@@ -1942,13 +1942,6 @@
     For: AllDays,            !- Field 2
     Until: 24:00,0.2;        !- Field 3
 
-  Schedule:Compact,
```

Comment:
I believe these are models provided by PNNL. They might want to make sure that they are making the same changes for their other prototype models too. @lymereJ do you know who to contact?

Reply:
Thanks! I'll let @jiangithub know.